python/lsst/synpipe/mergeOnlyFakes.py

```diff
@@ -26,7 +26,7 @@ class OnlyFakesMergeTask(mBand.MeasureMergedCoaddSourcesTask):
     # OnlyFakesMergeConfig to avoid having to fix overridden config parameters
     ConfigClass = mBand.MeasureMergedCoaddSourcesConfig
 
-    def run(self, patchRef):
+    def runDataRef(self, patchRef):
```

Comment:
I think this class also needs a `run` method so it can be used when you don't have a dataRef.

There are a few code style violations that are existing (e.g., `i_ss`, using docstrings instead of comments) that you would be welcome to clean up if you're able.

Reply:
This ticket will leave some Tasks without a `run` method. Refactoring all the Tasks was out of scope for this ticket.

Reply:
But the refactoring is not difficult, and it's required to meet the actual goal of the RFC.

Reply:
Sounds like we should just make sure those cleanup tickets are also conditions of calling the RFC implemented.  I don't think there's anything wrong with a `Task` that lacks a `run` method aside from the fact that this leaves the RFC implementation incomplete.

Reply:
This ticket is not final implementation ticket of RFC-352: Please see https://jira.lsstcorp.org/browse/DM-11098 and the comment therein which describes the relationship between 11098 and 2639.